src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -79,11 +79,19 @@ class PulseListener(object):
     """
     Listen to pulse messages and trigger new tasks
     """
-    def __init__(self, pulse_user, pulse_password, pulse_listener_hooks):
+    def __init__(self,
+                 pulse_user,
+                 pulse_password,
```

Comment:
Note: I find it strange that we need a Pulse user and password, given that Pulse message are public (e.g. visible on https://tools.taskcluster.net/pulse-inspector/ without authentication). Are we sending Pulse messages as well?